src/Profile.js

```diff
@@ -2,12 +2,14 @@ import React, { useState, useEffect } from "react";
 import Firestore from "./services/firestore";
 import { makeStyles } from "@material-ui/core/styles";
 import Card from "@material-ui/core/Card";
+import "./Profile.css";
 import CardActionArea from "@material-ui/core/CardActionArea";
 import CardActions from "@material-ui/core/CardActions";
 import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
 import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
+import plusSign from "./plusSign.png"
```

Comment:
I think you only need this import in the newCard component.

Reply:
true